pkg/remote/remote_test.go

```diff
@@ -325,7 +325,7 @@ func TestCreateDockerfile(t *testing.T) {
 				params: &Params{
 					BaseImage:           "test-image",
 					Manifest:            fakeManifest,
-					BuildEnvVars:        map[string]string{"OKTETO_BUIL_SVC_IMAGE": "ONE_VALUE", "OKTETO_BUIL_SVC2_IMAGE": "TWO_VALUE"},
+					BuildEnvVars:        map[string]string{"OKTETO_BUIL_SVC_IMAGE": "ONE_VALUE", "OKTETO_BUILD_SVC2_IMAGE": "TWO_VALUE"},
```

Comment:
I think you forgot a D here:
```suggestion
					BuildEnvVars:        map[string]string{"OKTETO_BUILD_SVC_IMAGE": "ONE_VALUE", "OKTETO_BUILD_SVC2_IMAGE": "TWO_VALUE"},
```

Reply:
I copied the typo from the existing test but it's already fixed in 39438f2469503d77697788d1ab0c478bc8a1f1c8